Add Chronograf documentation to 3.0 (#5039)
influxdata/docs-v2: content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/_index.md

> There are multiple ways to execute InfluxQL queries with {{< cloud-name >}}.
> Choose from the following options:

Reviewer: ```suggestion
Choose from the following options for executing InfluxQL queries with {{< cloud-name >}}:
```
